### PR TITLE
Added rainbow enchant t3

### PR DESCRIPTION
**Pull request by j10a1n15**
The book bundle was added, but the enchanted book itself not
image

### CODE REVIEW

File: items/RAINBOW;3.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "itemid": "minecraft:enchanted_book",
+    "displayname": "§f§fEnchanted Book",
+    "nbttag": "{ench: [],HideFlags: 254,display: {Lore: [\"§9Rainbow III\", \"§7Grants an §a80% §7chance to\", \"§7change the color of a sheep when\", \"§7shearing it, which will also\", \"§7flip it, of course.\", \"\", \"§7Applicable on: §9Shears\", \"§7Use this on an item in an Anvil\", \"§7to apply it!\", \"\", \"§f§lCOMMON\"],Name: \"§fEnchanted Book\"},ExtraAttributes: {id: \"ENCHANTED_BOOK\",enchantments: {rainbow: 3}}",
+    "damage": 0,
+    "lore": [
+      "§9Rainbow II",
```
Comment (jani270) [inline items/RAINBOW;3.json:7]: Rainbow II Moment

**Review by jani270 — CHANGES_REQUESTED** [review]
Please add it to the correct parent and format the file with the ingame save option again.

**Comment by jani270** [comment]
Oh and please change your branch to master.

**Comment by j10a1n15** [comment]
> Please add it to the correct parent and format the file with the ingame save option again.

I would love to format it, but I crash when trying to open the ingame editor

```
java.lang.NullPointerException: Updating screen events
	at io.github.moulberry.notenoughupdates.itemeditor.NEUItemEditor.<init>(NEUItemEditor.java:93)
	at io.github.moulberry.notenoughupdates.NEUOverlay.keyboardInput(NEUOverlay.java:1183)
	at io.github.moulberry.notenoughupdates.listener.RenderListener.onGuiScreenKeyboard(RenderListener.java:1136)```

**Comment by jani270** [comment]
> Oh and please change your branch to master.

.

**Comment by j10a1n15** [comment]
I dont really understand, so I switch my branch to master and then? Do I pr from the master branch or what? Whats the difference?

**Comment by jani270** [comment]
> I dont really understand, so I switch my branch to master and then? Do I pr from the master branch or what? Whats the difference?

Items that are new/changed should always be pr'ed into the master branch as the prerelease branch is only active for people who are using any 2.1.1 version. The prerelease branch will be updated after a few changes on the master

**Comment by j10a1n15** [comment]
I see, thank you!

**Comment by jani270** [comment]
Closing in favour of #991

**Pull request closed without merging**